except_star.md

```diff
@@ -199,14 +204,14 @@ except *OSerror as errors:
   for e in errors:
     if e.errno != errno.EPIPE:
        new_errors.append(e)
-  raise ExceptionGroup(*new_errors)
+  raise ExceptionGroup(e.msg, *new_errors)
```

Comment:
‘e.msg’ seems wrong here, e is a leftover from the loop.

Reply:
Right, should be errors.msg.